lib/index.js

```diff
@@ -57,6 +69,18 @@
   // Some properties that should not be overridden in the Text prototype.
   var doNotOverridePropertiesInTextNodes = ['textContent'];
 
+  // Some new properties that should be defined in the Text prototype.
+  var defineInTextNodes = ['assignedSlot'];
+
+  // Some properties that should not be overridden in the Comment prototype.
+  var doNotOverridePropertiesInCommNodes = ['textContent'];
+
+  // Some new properties that should be defined in the Comment prototype.
+  var defineInCommNodes = [];
+
+  // Nodes that should be slotted
+  var slottedNodeTypes = [1, 3];
```

Comment:
Might be nice to somehow reference that this is element and text nodes – maybe link to https://developer.mozilla.org/en/docs/Web/API/Node/nodeType in a comment, or better yet, using `Node.ELEMENT_NODE` / `Node.TEXT_NODE`